packages/framework/framework/tsconfig.json

```diff
@@ -1,23 +1,8 @@
 {
+  "extends": "@fluentui-react-native/scripts/tsconfig.json",
   "compilerOptions": {
-    "baseUrl": ".",
-    "target": "es5",
-    "jsx": "react",
     "outDir": "lib",
-    "declaration": true,
-    "sourceMap": true,
-    "experimentalDecorators": true,
-    "importHelpers": true,
-    "strictNullChecks": true,
-    "noImplicitAny": true,
```

Comment:
looks like several values are changing from what they were before. I guess any issues should show up in trying to build?

Reply:
That was my thought, but I can do a smaller sublet of changes. There's a whole lot more "combine tsconfigs" I can do in a separate PR

Reply:
quick glance tells me most of these are changing to be less strict settings so this particular change should be ok i think

Reply:
Oh. Less strict not good. I undid the ones so they stay overrides.